### PR TITLE
Portfolio-desktop

### PR DESCRIPTION
**Pull request by 28919926**
Add media queries to css file to make a desktop version of the project.
Modify form changes to fit the design of figma.
Make changes on HTML file.
Run linters and fix.

### CODE REVIEW

**Comment by BertrandConxy** [comment]
Hello @28919926, Please sure that all linters are passing 🙏

**Review by BertrandConxy — COMMENTED** [review]
Hi @28919926 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that has some linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -711,3 +735,279 @@ button:active {
...
+  list-style: none;
+  display: inline;
+  padding: 8px 12px;
+  position: relative;
+  justify-content: space-between;
+  text-decoration: none;
+  min-width: 375px;
+}
+
+@media screen and (min-width: 1500px) {
```
Comment (ghost) [inline style.css]: - [ ] Would you please use the required Desktop breakpoint in the CSS file `@media screen and (min-width: 768px`. It should not be more than `1440px` as anything above that would be an extra large screen.
- [ ]  Your `about me` cards appear to be aligned vertically. Would you please align them horizontally to match the Figma design  `Hint` Use `Flexbox`

### Your Design

image

### Figma design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @28919926 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
Hi @28919926 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 💯🌟 

5SW

## Status: APPROVED 🔥 🔥 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly remove the empty space at the bottom of the page
   
     - ### Your design
 
     image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**